src/components/HelloComponent.js

```diff
@@ -9,11 +11,29 @@ class HelloComponent extends Component {
 
   render() {
     return (
+     <div>
       <h1>
-        {customMessage.welcomeToBarefootNomad}
-      </h1>
+          {customMessage.welcomeToBarefootNomad}
+        </h1>
+        <h2>{this.props.message}</h2>
+        <button onClick= {this.props.displayMessage}>
+        try redux
+        </button>
+      </div>
     );
   }
 }
 
-export default HelloComponent;
+const mapStateToProps = state => {
+  return {
+   message : state.message
+  }
+};
+
+const mapDispatchToProps = dispatch => {
+  return {
+  displayMessage : () => dispatch({ ...firstMessage, value : `Redux is working !!!`})
```

Comment:
It seems this line is not covered by your tests. Could you take care of this?

![Screenshot from 2020-05-14 17-25-22 1 (1)](https://user-images.githubusercontent.com/45935765/81954289-f60a5b00-9608-11ea-81a0-3e208ee9e2fb.png)